This patch fixes g++ compilation errors due to missing function prototypes.

Signed-off-by: Lucas C. Villa Real <lucasvr@gobolinux.org>

--- vnc-4_1_2-unixsrc/common/network/TcpSocket.cxx.orig	2010-06-09 16:35:21.000000000 -0300
+++ vnc-4_1_2-unixsrc/common/network/TcpSocket.cxx	2010-06-09 16:35:36.000000000 -0300
@@ -31,6 +31,7 @@
 #include <netinet/tcp.h>
 #include <netdb.h>
 #include <unistd.h>
+#include <stdlib.h>
 #include <errno.h>
 #include <string.h>
 #include <signal.h>
--- /Data/Compile/Sources/vnc-4_1_2-unixsrc/unix/tx/TXImage.cxx.orig	2010-06-09 16:37:00.000000000 -0300
+++ /Data/Compile/Sources/vnc-4_1_2-unixsrc/unix/tx/TXImage.cxx	2010-06-09 16:37:15.000000000 -0300
@@ -21,6 +21,7 @@
 
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <strings.h>
 #include <sys/types.h>
 #include <sys/ipc.h>
--- /Data/Compile/Sources/vnc-4_1_2-unixsrc/unix/x0vncserver/Image.cxx.orig	2010-06-09 16:38:07.000000000 -0300
+++ /Data/Compile/Sources/vnc-4_1_2-unixsrc/unix/x0vncserver/Image.cxx	2010-06-09 16:38:14.000000000 -0300
@@ -21,6 +21,7 @@
 
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/types.h>
 #include <sys/ipc.h>
 #include <sys/shm.h>
